emf/loadflow_tool/model_merger/merge_functions.py

```diff
@@ -121,6 +140,10 @@ def create_sv_and_updated_ssh(merged_model, original_models, scenario_date, time
     ssh_data = load_opdm_data(original_models, "SSH")
     ssh_data = triplets.cgmes_tools.update_FullModel_from_filename(ssh_data)
 
+    # FIX BRELL LINES
+    ssh_data = set_brell_lines_to_zero(ssh_data)
```

Comment:
1. Fix needs to be done before LF
2. Only implement in RMM worker, this function is shared by RMM and CGM (create opdm object meta)